### ISSUE TITLE
[Feature Request] Allow omitting "az" prefix

### ISSUE THREAD

**Issue by lostintangent** [Microsoft/vscode-azurecli#7]
Since the scrapbook is specific to Azure CLI 2.0, it would be cool if you could omit the `az` prefix and just run commands without it (e.g. `webapp create` vs. `az webapp create`). Maybe that could be a setting that devs could opt-in to?

**Comment by chrmarti** [comment]
Currently you can type `web<enter>` and it will expand to `az webapp`.

**Comment by lostintangent** [comment]
@chrmarti Are you referring to the snippet? In my case, the completion list doesn't actually keep up with my typing, so if I type `web<enter>` it just enters a newline, instead of expanding the snippet. If I wait a second after typing `web`, then I see the behavior you mentioned, but it would still be cool to allow omitting the `az` prefix.

**Comment by chrmarti** [comment]
Yes, it should be faster on the second try. We can keep this issue to track your request.

**Comment by lostintangent** [comment]
Sounds good, thanks!
